src/queries/watchListQuery.ts

```diff
@@ -0,0 +1,35 @@
+import { gql } from "@apollo/client";
+import djs from "dayjs";
+
+const today = djs().format("YYYY-MM-DD");
```

Comment:
if we find ourselves using this query in other places we could pull this out and just pass it as an argument when we make the query